### PR TITLE
Issue #603: Refactor EmailSettings attributes

### PR DESCRIPTION
**Pull request by raluka**
EmailSetting incosistency #603

### CODE REVIEW

**Comment by differentreality** [comment]
Since we are at it, what about we also uniform the name of the variables?
Like:
call_for_papers_dates_updates_body -> call_for_papers_dates_updated_body
venue_update_body -> venue_updated_body
updated_conference_dates_body -> conference_dates_updated_body
updated_conference_registration_dates_body -> conference_registration_dates_updated_body

What do you think?

File: db/schema.rb [hunk 1]
```diff
@@ -515,7 +515,7 @@
     t.integer  "photo_file_size"
     t.datetime "photo_updated_at"
     t.string   "street"
-    t.integer  "postalcode"
+    t.string   "postalcode"
```
Comment (differentreality) [inline db/schema.rb]: That is not relevant to this PR

**Comment by hennevogel** [comment]
I very much like the consistency!

**Comment by raluka** [comment]
@differentreality , @hennevogel what do you think about this PR? Should I improve it  more?

**Comment by differentreality** [comment]
It looks good, but letter_opener does not work (for me?) when it comes to emails for things happening outside of devise. Should it be working? I would have liked to try this PR out.

**Comment by raluka** [comment]
@differentreality, what exactly did not work?

**Comment by differentreality** [comment]
@raluka when you accept a proposal, for example, there is an email sent to the submitter (assuming the email settings params are set). The same happens when you reject a proposal, or when you register to a conference etc. 
I don't see that email being sent to me through letter_opener, neither in master nor in your PR. So even though everything seems nice, I can't really try it out in practice.

**Comment by raluka** [comment]
Checkboxes for sending notifications are disabled by default, they have to be enabled  in order to send emails for updates.

**Comment by differentreality** [comment]
@raluka yes I know. So?

**Comment by raluka** [comment]
Also, start rake jobs:work. With these 2 conditions, sending notifications works for me.

**Comment by hennevogel** [comment]
So let's get this in, thanks :girl:!

**Pull request merged**